src/ui/components/diary/FeelingContainer.tsx

```diff
@@ -17,15 +18,16 @@ export function FeelingContainer({ diary, onClick }: FeelingContainerProps) {
       </Typography>
       <FeelingCatList>
         <>
-          {FEELING_CAT_TYPE.map((el, index) => {
+          {FEELING_CAT_TYPE.map((el) => {
```

Comment:
P3:

```suggestion
          {FEELING_CAT_TYPE.map((feelingCat) => {
```

el 보다는 의미있는 네이밍을 해주면 어떨까요?

Reply:
프로젝트에서 검색해보니 el을 사용한 부분이 여러곳 보이네요. 이 부분 유의미한 네이밍으로 수정하겠습니다~!